### PR TITLE
Fix format string in ce.c

### PR DESCRIPTION
**Pull request by ryabouza**
Format string type descriptor was incorrect, causing build failures on certain machines.

### CODE REVIEW

File: ce.c [hunk 1]
```diff
@@ -297,9 +297,9 @@ bool ce_insert_line(Buffer* buffer, int64_t line, const char* string)
      char** new_lines = malloc(new_line_count * sizeof(char*));
      if(!new_lines){
           if(buffer == g_message_buffer){
-               printf("%s() failed to malloc new lines: %ld\n", __FUNCTION__, new_line_count);
```
Comment (justy989) [inline ce.c:421]: I used %ld everywhere when printing int64_ts, is it really wrong? I wasn't able to compile unless I used %ld, I'm not sure I tried %lld though.

**Comment by ryabouza** [comment]
Yeah, it won't compile on Mac unless it's lld. There's more places I'm fixing. Hold on...

**Comment by ryabouza** [comment]
I had to make a small Makefile change also.  Things build for me now :)

File: Makefile [hunk 2]
```diff
@@ -3,7 +3,7 @@ CFLAGS=-Wall -Werror -Wextra -std=gnu11 -ggdb3
 LINK=-lncurses
 
 ce: main.c ce.o ce_config.so
-	$(CC) $(CFLAGS) $^ -o $@ $(LINK) -ldl -Wl,-rpath=.
+	$(CC) $(CFLAGS) $^ -o $@ $(LINK) -ldl -Wl,-rpath,.
```
Comment (robber-m) [inline Makefile:6]: Did you make sure that this will work for us on linux machines too?

Comment (ryabouza) [reply Makefile:6]: Working on that now

Comment (ryabouza) [reply Makefile:6]: This works on linux/mac

**Comment by ryabouza** [comment]
Okay, this should be ready to go.

**Review by robber-m — APPROVED** [review]

**Pull request merged**